2_adventures/001/jonkernpa/04_day/README.md

```diff
@@ -0,0 +1,11 @@
+# Adventure Day 4
+
+Just what is an adventure?
+
+How lame... spent an hour cleaning up my lousy mess of git rebase hell.
+
+     git remote add upstream /url/to/original/repo
+     git fetch upstream
+     git checkout master
+     git reset --hard upstream/master
+     git push origin master --force
```

Comment:
Yes, `git pull upstream master` will do both a fetch and merge operation from the 'central' repository to your repository..

But before that step, you likely want to do `git checkout master` so that you are on the master branch.

This sounds right.